Underanalyzer/VMConstants.cs

```diff
@@ -1,11 +1,19 @@
-﻿namespace Underanalyzer;
+﻿/*
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+namespace Underanalyzer;
 
 /// <summary>
 /// Contains constant values used by the GameMaker VM.
 /// </summary>
 internal static class VMConstants
 {
-    // Function names used for try..catch..finally statements
+    // TODO: these are public, give them summary tags or private them.
```

Comment:
This class is internal (and are just constants)

Reply:
ack shouldve looked at the modifier whoops